tests/UsedClassesAvailableTest.php

```diff
@@ -12,26 +12,16 @@ class UsedClassesAvailableTest extends TestCase
 
     public function testClassesAreInstantiable(): void
     {
-        $namespace = str_replace('Tests', '', __NAMESPACE__);
+        $files = $this->getPluginClasses();
 
-        foreach ($this->getPluginClasses() as $class) {
-            $classRelativePath = str_replace(['.php', '/'], ['', '\\'], $class->getRelativePathname());
-
-            $this->getMockBuilder($namespace . '\\' . $classRelativePath)
-                ->disableOriginalConstructor()
-                ->getMock();
-        }
-
-        // Nothing broke so far, classes seem to be instantiable
-        $this->assertTrue(true);
+        static::assertCount(1, $files);
```

Comment:
now you only check, if the classes are present. before there was a check if they were instantiatable

Reply:
there is only 1 twig file... no php files